pom.xml

```diff
@@ -132,10 +132,24 @@
         <version>1.2.3</version>
         <scope>test</scope>
       </dependency>
+
+      <!-- Testkit Dependencies -->
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-core</artifactId>
+        <version>2.11.3</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>2.11.3</version>
+      </dependency>
```

Comment:
Do we need Jackson for the driver it self? Because, if it's only needed for the test kit, it should be only on the test kit project.

Reply:
My mistake, it's just the dependency management .